namespaces/Representor.md

```diff
@@ -0,0 +1,133 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Representor (Object Element)
+
+The Representor represents a Hypermedia Resource, with it's available links, transitions and attributes.
+
+### Properties
+
+- `links` - (Array Element of Link) - The links from this resource.
+- `transitions` - (Array Element of Transition) - The available transitions from this resource.
+- `representors` - (Array Element of Representor) - The embedded links (resources) from this resource.
+- `attributes` - (Object Element) - Any properties of the resource.
+- `metadata` - (Object Element) _ Meta Attributes, for example, may include a `title` and `description`.
+
+## Link (Object Element)
+
+A link element representing a link and it's relation from the Representor.
+
+### Properties
+
+- `relation` - (String Element, required) - The relation from the Representor to this link.
+- `uri` - (String Element, required) - The URI for this link.
+
+### Example
+
+```json
+{
+    "element": "object",
+    "content": [
```

Comment:
Oh I see it now, but this isn't quite what is written in MSON on line 23 (you are missing the `content` property there